docs/regulatory-framework/governance-framework.md

```diff
@@ -17,22 +19,35 @@ To easily grasp the layers of our Governance Framework, think of flight level as
 
 ## 30,000ft - Data Space Level
 
-At this level, the Governance Framework is defined by the Catena-X association and serves as the basis for the certification of Core Service Providers (CSP) A and B. The "Golden Rules" document, which outlines the basic principles of the data ecosystem, is published by the Catena-X association and provides clear guidelines for necessary specifications at lower levels. It also specifies how far a technical standard can go. The agreement between the CSPs and participants (e.g., data providers and data consumers) is established during onboarding.
+At this level, the governance framework is defined by the Catena-X Association and the basic principles of the data ecosystem are outlined. Compliance is mandatory for all participants in the data space.
+There are several normative documents to consider.
+
+The first is the operating model. The operating model lays the foundation and is the pre-requisite for further standardization, certification, implementation, and operations of software components and partners in the Catena-X data space. The operating model is continuously updated to reflect the evolution of business and regulatory requirements, as well as emerging technologies.
+
+The "10 Golden Rules" document, which outlines the basic principles of the data ecosystem, is published by the Catena-X association and provides clear guidelines for necessary specifications at lower levels. It also specifies how far a technical standard can go. The agreement between the CSPs and participants (e.g., data providers and data consumers) is established during onboarding.
+
+The third document is the Data Exchange Governance. This Data Exchange Governance sets out the key principles for data exchange that apply to each Participant when exchanging data in the Catena-X Data Space for one or more purposes as predefined by the Association ("Predefined Purpose(s)"). Each Participant must agree to this Data Exchange Governance in order to effect data exchanges through a registered connector. The process of agreeing to this Data Exchange Governance, as well as for concluding legally binding data exchange contracts is not part of this Data Exchange Governance. It does not determine or restrict how Participants need to make their contractual declarations legally binding. Rather, the Data Exchange Governance contains standardised technical parameters for data exchange and other mandatory requirements that correspond to the predefined purposes described in more detail under the following link (‘Repository’): https://github.com/catenax-eV/cx-odrl-profile.
 
 ## 20,000ft - Use Case Level
```

Comment:
The rework of 20,000 is missing - there are still all the Use-Case Framework agreements: 
![image](https://github.com/user-attachments/assets/82d76d04-4252-4b8f-bcc7-7e4e758a49b4)

Reply:
Will there still be content for 20,000ft after that or will this section be empty?